docs/index.md

```diff
@@ -0,0 +1,18 @@
+# Welcome to outpack-py's documentation!
+
+```{toctree}
+:maxdepth: 2
+:caption: 'Contents:'
+
+readme
+code
+```
+
+% uses MyST markdown format https://jupyterbook.org/en/stable/content/myst.html
```

Comment:
if we're using ReST :vomiting_face:  the extension should be .rst not .md

Reply:
Looks like this is using MyST, which sounds like it is slightly different? And they use `.md` in their docs. https://myst-parser.readthedocs.io/en/latest/intro.html#write-a-markdown-document

Reply:
yep MyST is all .md files